src/Messages/OWSMessageSender.m

```diff
@@ -257,7 +257,7 @@ @interface OWSMessageSender ()
 @property (nonatomic, readonly) id<ContactsManagerProtocol> contactsManager;
 @property (nonatomic, readonly) ContactsUpdater *contactsUpdater;
 @property (nonatomic, readonly) OWSDisappearingMessagesJob *disappearingMessagesJob;
-@property (nonatomic, readonly) NSOperationQueue *sendingQueue;
+@property (atomic, readonly) NSMutableDictionary *sendingQueueMap;
```

Comment:
Annotate: done.

atomic: Although its read-only access today, I like use atomic for any property that might be accessed from multiple threads.  It's an attempt to improve readability.  I might think twice about it if this was a perf hotspot, but it's very far from being that.